ansible/roles/debops.postfix/defaults/main.yml

```diff
@@ -256,6 +271,351 @@ postfix__tls_dh512_param_file: '{{ ansible_local.dhparam[postfix__dhparam_set]
+                                                                   # ]]]
+# .. envvar:: postfix__ldap_default_virtual_domain_rdn [[[
+#
+# The rdn of the default virtual mail domain used by Postfix.
+# This is usually the same domain used by the LDAP.
+postfix__ldap_default_virtual_domain_rdn: '{{ ansible_domain }}'
+                                                                   # ]]]
+# .. envvar:: postfix__ldap_default_virtual_domain_dn [[[
+#
+# The dn of the default virtual mail domain used by Postfix.
+postfix__ldap_default_virtual_domain_dn: '{{ (["dc=" + postfix__ldap_default_virtual_domain_rdn]
+                                              + postfix__ldap_domains_dn) | join(",") }}'
+                                                                   # ]]]
+                                                                   # ]]]
+# LDAP connection options [[[
+# ^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+# .. envvar:: postfix__ldap_server_uri [[[
+#
+# The URI address of the LDAP server used by Postfix.
+postfix__ldap_server_uri: '{{ (ansible_local.ldap.uri
+                                if (ansible_local|d() and ansible_local.ldap|d() and
+                                    ansible_local.ldap.uri|d())
+                                else [""]) | first }}'
+
+                                                                   # ]]]
+# .. envvar:: postfix__ldap_server_port [[[
+#
+# The TCP port which should be used for connections to the LDAP server.
+postfix__ldap_server_port: '{{ ansible_local.ldap.port
+                                if (ansible_local|d() and ansible_local.ldap|d() and
+                                    ansible_local.ldap.port|d())
+                                else ("389" if postfix__ldap_start_tls|bool else "636") }}'
+
+                                                                   # ]]]
+# .. envvar:: postfix__ldap_start_tls [[[
+#
+# If ``True``, Postfix will use STARTTLS extension to make encrypted
+# connections to the LDAP server.
+postfix__ldap_start_tls: '{{ ansible_local.ldap.start_tls
+                              if (ansible_local|d() and ansible_local.ldap|d() and
+                                  (ansible_local.ldap.start_tls|d())|bool)
+                              else True }}'
+
+                                                                   # ]]]
+                                                                   # ]]]
+
+# LDAP settings [[[
+# ^^^^^^^^^^^^^^^^^
+
+# .. envvar:: postfix__ldap_user_filter [[[
+#
+# The LDAP filter used to look up user accounts in the directory.
+postfix__ldap_user_filter: '(&
+                               (objectClass=inetOrgPerson)
+                               (objectClass=postfixUser)
+                             )
+                             (mailEnabled=TRUE)
+                             (|
+                               (authorizedService=postfix)
```

Comment:
I think that going with `smtpd` here instead of `postfix` would make this value more universal, since other DebOps roles that manage SMTP services could then use that filter value as well.

Reply:
Agree